parser.rb

```diff
@@ -0,0 +1,43 @@
+
+class Parser
+  def initialize(filename)
+    @file = File.open(filename, "r")
+    @text = ''
+    @file.each_line do |txt|
+      @text = @text + txt
+    end
+  end
+
+  def array_of_words
+    @text.split(/\W+/)
```

Comment:
The first part had me excited at first, but in fact that's going to give trouble with the single quotes in the original passage, which you would want to cut out.  Ultimately, you'd need to split on a regex that finds non-word-characters and single-quotes-that-aren't-between-other-word-characters.